src/vm/wren_utils.c

```diff
@@ -91,7 +91,7 @@ static void wrenSymbolTableGrow(WrenVM *vm, SymbolTable *symbols)
     wrenSymbolTableClearBitset(symbols);
 
     size_t oldCap = symbols->hCapacity;
-    symbols->hCapacity *= 2;
+    symbols->hCapacity = wrenPowerOf2Ceil(symbols->hCapacity);
```

Comment:
This change is broken: `wrenPowerOf2Ceil` does not return the next power of 2 if the current number is a power of 2. https://godbolt.org/z/P3vGnd

Reply:
Oh good catch. So `wrenPowerOf2Ceil(symbols->hCapacity + 1)` would work. I think *= 2 is simpler, I call `wrenPowerOf2Ceil()` simply for consistency but I feel like it might make it harder to read